core-services/nvidia_gpu.tf

```diff
@@ -0,0 +1,80 @@
+resource "kubernetes_daemonset" "nvidia" {
+  count = try(var.node_groups.additional_node_groups.0.compute_type == "GPU" ? 1 : 0, 0)
+  metadata {
+    name      = "nvidia-device-plugin-daemonset"
+    namespace = kubernetes_namespace_v1.md-core-services.metadata.0.name
+    labels = {
+      k8s-app = "nvidia-device-plugin-daemonset"
+    }
+  }
+  spec {
+    selector {
+      match_labels = {
+        name = "nvidia-device-plugin-ds"
+      }
+    }
+    strategy {
+      type = "RollingUpdate"
+    }
+    template {
+      metadata {
+        labels = {
```

Comment:
Same with pod labels. Without this, the pods won't be monitored.

Reply:
Added